templates/projects/new.tmpl

```diff
@@ -50,7 +50,7 @@
 				<div class="ui divider"></div>
 				<div class="ui left">
 					{{if .PageIsEditProjects}}
-					<a class="ui cancel button" href="{{.RepoLink}}/projects">
+					<a class="ui cancel button" href="/{{.Org.Name}}/-/projects{{if eq .redirect "project"}}/{{.projectID}}{{end}}">
```

Comment:
Wait, this is incorrect.

The "AppSubUrl" is missing.

Reply:
I think `$.HomeLink` is what you need. Same usage at line 6.